package.json

```diff
@@ -28,18 +28,18 @@
     "tslint": "tslint \"app/**/*.ts\""
   },
   "dependencies": {
-    "@angular/common": "2.0.0",
-    "@angular/compiler": "2.0.0",
-    "@angular/core": "2.0.0",
-    "@angular/forms": "2.0.0",
-    "@angular/http": "2.0.0",
-    "@angular/platform-browser": "2.0.0",
-    "@angular/platform-browser-dynamic": "2.0.0",
-    "@angular/platform-server": "2.0.0",
-    "@angular/router": "3.0.0",
+    "@angular/common": "2.0.1",
+    "@angular/compiler": "2.0.1",
+    "@angular/core": "2.0.1",
+    "@angular/forms": "2.0.1",
+    "@angular/http": "2.0.1",
+    "@angular/platform-browser": "2.0.1",
+    "@angular/platform-browser-dynamic": "2.0.1",
+    "@angular/platform-server": "2.0.1",
+    "@angular/router": "3.0.1",
     "email-validator": "1.0.4",
     "everlive-sdk": "1.7.1",
-    "nativescript-angular": "^1.0.0",
+    "nativescript-angular": "^1.0.1",
```

Comment:
@hdeshev I got an error when testing this because it looks like nativescript-angular 1.0.1 isn’t on npm quite yet.